setup.py

```diff
@@ -68,34 +67,31 @@ def package_files(directory, strip_leading):
               'pytest-cov',
               'codecov'
           ],
+          'ci': ['codecov'],
```

Comment:
This in already in 'pi' and for the PC installation part of the Conda env, so we don't need this here any longer.

Reply:
I don't want to rely on `conda` too much. I want to support traditional `pip` installs as well.

Reply:
Yes, this will work too, as in setup.py we more or less duplicate what we have in the conda yaml files. So if you build your env with conda and then run `pip install -e .` it will basically do nothing b/c everything is there and b/c dependencies are set up correctly, you also don't need to upgrade `numpy` etc. 
However, if you only install with `pip` using setup, your installation will also work, but I thought this is only really relevant for PI / Nano where we don't have conda. But it should work on every other box, too, I agree. For this we would need `tf` in here too, the way you added it. Hopefully it doesn't mess up the Conda env, I can check that.

Reply:
Yeah shouldn't be a problem. Sorry put this whole shebang one line too high...